converter/src/test/java/gov/cms/qpp/conversion/decode/QualityMeasureIdDecoderTest.java

```diff
@@ -5,18 +5,18 @@
 import gov.cms.qpp.conversion.xml.XmlException;
 import gov.cms.qpp.conversion.xml.XmlUtils;
 import org.jdom2.Element;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeEach;
 
 import static com.google.common.truth.Truth.assertWithMessage;
 
 /**
  * Test for the QualityMeasureIdDecoder
  */
-public class QualityMeasureIdDecoderTest {
+class QualityMeasureIdDecoderTest {
 	private QualityMeasureIdDecoder objectUnderTest;
 
-	@Before
+	@BeforeEach
 	public void setup() {
```

Comment:
Any reason you kept the `public` on this function?

Reply:
Missed it. Thanks!